src/Mailer/SMTP.php

```diff
@@ -4,11 +4,39 @@
 
 namespace Atk4\Outbox\Mailer;
 
+use PHPMailer\PHPMailer\SMTP as PHPMailerSMTP;
+
 class SMTP extends AbstractMailer
```

Comment:
should be named `Smtp` to be consistent with other naming

Reply:
done!